modules/admin_manual/pages/installation/ubuntu_18_04.adoc

```diff
@@ -20,8 +20,7 @@ To do so, follow the instructions below:
 
 [source,console,subs="attributes+"]
 ----
-apt install software-properties-common && \
-  apt update && \
+apt update && \
```

Comment:
why does the text above say "and that PHP 7.3 is available in the APT repository"?

I thought that Ubuntu 18.04 came with PHP 7.2 and that you need to do the "special composer repository stuff" to get PHP 7.3 from the `ondrej` repo? But maybe I am now wrong - I have not done an "out of the box" Ubuntu 18.04 install for a while.

Reply:
I was planning to correct the information in a follow-up PR as the original change came from a contributor's PR.